### PR TITLE
fix: #897 frontend minor style changes

### LINKED ISSUE #897: Frontend minor adjustment
**Description**
Some small frontend adjustment make the UI looks better

**Acceptance Criteria**
- [x] Check and update the font size on the manage permission page to match the design, for example the font size of the menu bar is 12, but it's 14 in the UI mockup

  <br />

- [x] Confirm with @OlgaLiber2 and Alison, if this "Add user permission" button looks too large? And its border-radius will show some tiny space around, do we want to get rid of the border-radius for this button? 
Update: keep the button size, remove the border-radius for this specific button
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBc2UzIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--260719b5f58d2e0fdc490cf77d52b6d0f1a59b94/Screen%20Shot%202023-09-19%20at%204.44.20%20PM.png)

    <br />

- [x] Confirm with @OlgaLiber2 and Alison, if the height of the table header is too much? maybe we can reduce the height a bit?
Update: no need to change the header height
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBc0czIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--832e50c64ad0abe465974088c8a1fcf03a8aecc7/Screen%20Shot%202023-09-19%20at%204.42.39%20PM.png)
    <br />

- [x] The search input has a tiny space below (maybe no need to adjust this if we reduce the "Add user permission" button size)
Update: solved in Jota's pr #913
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBc1czIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--0fd7c6014ded634f2a25b672537aa1bdd4411340/Screen%20Shot%202023-09-19%20at%204.43.43%20PM.png)

     <br />

- [x] When click on the "Add user permission" button, the border effect doesn't look right
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBciszIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--32feab908bdb78060719b065c80babfcc1e2c219/image.png)

### PR DESCRIPTION
**Pull request by MCatherine1994**
Fix #897:
- update fontsize in side nav, and change the unit from px to rem
- remove the border radius for the speific add user permission button
- fix user data table background height problem
- update to use the latest nr-theme package
- remove unused class name

Style changes for button on focus is in the shared stylesheet

### CODE REVIEW

File: frontend/src/components/managePermissions/UserDataTable.vue [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 import InputText from 'primevue/inputtext';
-import { IconSize } from '@/enum/IconEnum';
 import { useConfirm } from 'primevue/useconfirm';
 import ConfirmDialog from 'primevue/confirmdialog';
-import type { PropType } from 'vue';
+
+import router from '@/router';
+import { IconSize } from '@/enum/IconEnum';
+import IconCapitol from '@/components/common/IconCapitol.vue';
+import Button from '@/components/common/Button.vue';
```
Comment (MCatherine1994) [inline frontend/src/components/managePermissions/UserDataTable.vue:15]: a lot of changes in this file are just auto formatting, the real changes mainly at the end of the file about styling

**Review by J0taFerreira — DISMISSED** [review]
everything is working fine!

**Comment by ianliuwk1019** [comment]
Hi Catherine, thanks for resolving these minor adjustment we discussed last time.
Maybe this one can adjust a little more (not sure if it is my local issue):
image

**Comment by MCatherine1994** [comment]
> Hi Catherine, thanks for resolving these minor adjustment we discussed last time. Maybe this one can adjust a little more (not sure if it is my local issue): image
> 
> May also reduce table margin to the top of background to be equal. Thanks.: image

Hi Ian, the input height fix is done in the shared stylesheet, will publish a new version and import again later. The table margin is like that in the design, but I saw next version the margin will be reduced when we add tap, maybe we can change when work on the next version.

**Comment by ianliuwk1019** [comment]
> Hi Catherine, thanks for resolving these minor adjustment we discussed last time. Maybe this one can adjust a little more (not sure if it is my local issue): image
> 
> May also reduce table margin to the top of background to be equal. Thanks.: image

I see, I am confused what style in where but other than that sounds good. Thanks.

**Review by ianliuwk1019 — APPROVED** [review]

**Review by ianliuwk1019 — APPROVED** [review]

**Comment by MCatherine1994** [comment]
We still left some minor style changes (button on focus effect, input height) in the common theme repo, but those are really tiny ones, we could wait to release the new theme version later.

**Review by ianliuwk1019 — APPROVED** [review]

**Pull request merged**